doc/tags/types.rst

```diff
@@ -0,0 +1,30 @@
+``types``
+=======
+
+The ``types`` tag declares the types for one or more names variables.
+
+To do this, specify a :ref:`mapping <twig-expressions>` of names to their types as string constants. 
+
+Here is how you declare that variable ``foo`` is a boolean, while ``bar`` is an integer (see note below):
```

Comment:
```suggestion
Here is how you declare that variable ``foo`` is a ``boolean``, while ``bar`` is an ``integer``:
```

Reply:
See my other comment below about defining a minimal set of types that must be supported.